src/main/java/com/sourcery/gymapp/backend/workout/exception/ErrorCode.java

```diff
@@ -7,6 +7,7 @@
 @RequiredArgsConstructor
 public enum ErrorCode {
     ROUTINE_NOT_FOUND("Routine not found"),
+    EXERCISE_NOT_FOUND("Exercise not found"),
     INTERNAL_SERVER_ERROR("Internal server error"),
     USER_NOT_FOUND("User not found"),
     USER_NOT_AUTHENTICATED("USER_NOT_AUTHENTICATED");
```

Comment:
Change `USER_NOT_AUTHENTICATED("USER_NOT_AUTHENTICATED");` to `USER_NOT_AUTHENTICATED("user not authentificated");`

Reply:
Okay, that's not a part of my task, but i can update it :)